src/mobility/queries/use-value-code-query.tsx

```diff
@@ -2,11 +2,14 @@ import {useQuery} from '@tanstack/react-query';
 import {getValueCode} from '@atb/mobility/api/api';
 import {useAuthState} from '@atb/auth';
 
-export const useValueCodeQuery = (operatorId: string | undefined) => {
+export const useValueCodeQuery = (
+  operatorId: string | undefined,
+  enabled: boolean,
+) => {
   const {userId, authStatus} = useAuthState();
   return useQuery({
     queryKey: ['mobilityValueCode', userId, operatorId],
     queryFn: () => getValueCode(operatorId),
```

Comment:
@marius-at-atb just wondering, should we use `useMutation` here instead of `useQuery`, since it has a `mutationAsync` that can be used, instead of calling `refetch` on the `useQuery`.

Reply:
Yes. Actually @gorandalum also suggested this before leaving for the day.

Reply:
Update: https://github.com/AtB-AS/mittatb-app/pull/4569/commits/997bbfa20d53afcdca2c6f0495383aba1b0cdd24